examples/internal/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -2037,6 +2037,8 @@
     },
     "/v2/example/empty": {
       "get": {
+        "summary": "Create a new ABitOfEverything",
+        "description": "This API creates a new ABitOfEverything",
```

Comment:
This comment is not correct either. There should not be a comment here. See https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/internal/proto/examplepb/a_bit_of_everything.proto#L541-L547